File: EZPZproject/apiEZPZ/src/main/java/com/example/apiezpz/auth/dto/SignUpRequest.java
package com.example.apiezpz.auth.dto;
import lombok.Data;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Data
public class SignUpRequest {
    private String username;
    private String password;
    private String name;
    private String phone;
    private String email;
    private String address;
    private String gender;
}

File: apiEZPZ/src/main/kotlin/com/example/apiezpz/auth/dto/SignUpRequest.kt
package com.example.apiezpz.auth.dto

data class SignUpRequest(
    var username: String = "",
    var password: String = "",
    var name: String = "",
    var phone: String = "",
    var email: String = "",
    var address: String = "",
    var gender: String = ""
) 